cmd_root.go

```diff
@@ -56,6 +56,8 @@ about your Continuous Integration builds.`,
 			prov = providerAzurePipelines
 		} else if _, present := os.LookupEnv("GITHUB_ACTIONS"); present {
 			prov = providerGitHubActions
+		} else if _, present := os.LookupEnv("BITBUCKET_BUILD_NUMBER"); present {
```

Comment:
bitbucket pipelines doesn't have any variable that is set `true` on each pipeline other than `CI` variable which is common in many CI systems like github-actions. So, I have used the build number to identify the integration system 
more info - https://support.atlassian.com/bitbucket-cloud/docs/variables-and-secrets/